web/app/js/components/util/ApiHelpers.jsx

```diff
@@ -89,6 +89,9 @@ export const ApiHelpers = (pathPrefix, defaultMetricsWindow = '1m') => {
   };
 
   const urlsForResource = {
+    "all": {
+      url: genResourceUrl("all")
```

Comment:
oof, does this function exist solely because `replication_controller` != `replicationcontroller`? ideally we could standardize, but short of that, perhaps we could simplify the function just a bit to better call out the situation?

```js
const urlsForResource = type => {
  if type === "replication_controller" {
    type = "replicationcontroller"
  }
  return namespace => {
    let baseUrl = '/api/stat?resource_type=' + type;
    return {
      rollup: !namespace ? baseUrl : baseUrl + '&namespace=' + namespace
    };
  };
};
```

Reply:
haha yes it does.  I was also considering having a version of `CanonicalKubernetesNameFromFriendlyName` on the frontend so we can stop dealing with this everywhere, but maybe that can be for later. 

yeah I can simplify this function!